lib/Migrator.php

```diff
@@ -57,17 +57,13 @@ public function initialize()
      * If $to is 0 then all migrations will be reverted.
      * If $to is null then all migrations will be executed.
      *
-     * @param string $to Version to run until
+     * @param string|null $to Version to run until
      * @param OutputInterface $output
      *
      * @return VersionInterface[] Executed migrations
      */
-    public function migrate($to = null, OutputInterface $output)
+    public function migrate($to, OutputInterface $output)
     {
-        if (false === $to) {
-            return array();
-        }
```

Comment:
isn't that a BC break? its odd usage of the parameter but $to can be anything, so it could be false too